src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -255,6 +269,40 @@ private void readRcaEnabledFromConf() {
         });
   }
 
+  /**
+   * Reads the mutedRCAList value from the rca.conf file, performs validation on the param value
+   * provided and on successful validation, updates the AnalysisGraph with muted RCA value.
+   *
+   */
+  private void readAndUpdateMutesRcas() {
```

Comment:
I am thinking if we should move this [here](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/3267f003f4b5acc70f8c603472cc1ed44859a310/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/scheduler/RCASchedulerTask.java#L374) ? `RCASchedulerTask` is the component that actually deals with the Graph.

Reply:
The other place would be [here](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/3267f003f4b5acc70f8c603472cc1ed44859a310/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java#L210). After we check if the RcaEnabled config change, we check if the rca.conf has been updated (by checking the last modified time). If so, then we check if the muted Rca list has been updated and if so, we do validate and update the muted RCA list. If it hasn't we move on, until next time. What do you think ?